app/src/main/java/com/io/moviO/MainActivity.kt

```diff
@@ -11,12 +11,10 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
-        val view = binding.root
-        setContentView(view)
+        setContentView(binding.root)
 
-        val movieDetails = MovieDetailsFragment()
         val tr = supportFragmentManager.beginTransaction()
```

Comment:
you can chain all those methods and remove the val

Reply:
resolved